charts/prometheus/Chart.yaml

```diff
@@ -0,0 +1,14 @@
+name: prometheus
+version: 8.0.0
```

Comment:
Since this is our specific fork, can we make the version zenko specific, like 8.0.0-z-pre-ga or something like this.

Reply:
`8.0.0-Z`